infinispan-remote/infinispan-secured-keycloak/docker-compose.yaml

```diff
@@ -0,0 +1,34 @@
+version: '3'
+
+networks:
+  mynetwork:
+    driver: bridge
+services:
+  infinispan-server:
+    image: quay.io/infinispan-test/server:14.0.x
+    volumes:
+      - ${PWD}:/user-config
```

Comment:
If we use `- ./infinispan-token-realm.yaml:/user-config/infinispan-token-realm.yaml` then we can call `docker-compose` from any dir.